tests/flow/test_undo_log.py

```diff
@@ -0,0 +1,286 @@
+from RLTest import Env
```

Comment:
Missing tests where multiple changes occur in a single query
For example a single query which performs multiple operations create, update, delete

most tests focus on a single change, there's no mixture of operations